README.md

```diff
@@ -28,7 +28,18 @@ The API reference is located at [realm.io/docs/java/api](http://realm.io/docs/ja
 
 ## Using Snapshots
 
-If you want to test recent bugfixes or features that have not been packaged in an official release yet, you can use a **-SNAPSHOT** release of the current development version of Realm via Gradle, available on [JFrog](http://oss.jfrog.org/oss-snapshot-local/io/realm/realm-android/)
+If you want to test recent bugfixes or features that have not been packaged in an official release yet, you can use a **-SNAPSHOT** release of the current development version of Realm via Gradle, available on [OJO](http://oss.jfrog.org/oss-snapshot-local/io/realm/realm-android/)
+
+    buildscript {
+        repositories {
+            maven {
+                url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
+            }
+        }
+        dependencies {
+            classpath "io.realm:gradle-plugin:<version>-SNAPSHOT"
```

Comment:
I really hate when other libraries do this because then you have to scramble around trying to figure out what is the latest version. Our current approach does take maintenance, sure, but it also makes the docs really easy to read. If we do this we should at least add a link to the `version.txt` as well.

A better approach could be to extend the release plugin so it automatically updated the version here as well? (No need in this PR though)